src/pages/Feedback.tsx

```diff
@@ -9,8 +9,8 @@ export const Feedback: React.FC = () => {
     {
       icon: <Mail className="w-8 h-8 text-indigo-600" />,
       title: 'Email',
-      value: 'support@kingdomchroniclesug.com',
```

Comment:
@isingoma  Every MR or PR should be exactly atomic , meaning let it speak and do just one thing you have described , no point in doing other things that dont speak to your branch naming